challenges/classes.js

```diff
@@ -1,7 +1,51 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
-// Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+class CuboidMaker {
+  constructor(attr) {
+    this.length = attr.length;
+    this.width = attr.width;
+    this.height = attr.height;
+  }
+  // Method goes here:
+  volume() {
+    return (this.length * this.width * this.height);
+  } // End of volume method
+  surfaceArea() {
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+  } // End of surface method
+}
+
+class CubeMaker extends CuboidMaker {
+  constructor(cubeAttr) {
+    super(cubeAttr);
+    this.length = cubeAttr.length;
+    this.width = cubeAttr.width;
+    this.height = cubeAttr.height;
+  }
+  // Method goes here:
+  volume() {
+    return (this.length * this.height * this.width);
+  } // End of voume method
+  surface() {
+    return (Math.pow(this.length, 2) * 6);
+  }
+}
+
+const cuboid = new CuboidMaker({
+  'length': 4,
+  'width': 5,
+  'height': 5
```

Comment:
well done! I like your use of comments. It makes your code very easy to read and understand.